src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -47,7 +47,7 @@ open class Publisher(private val topicName: String) {
         this.pubsubPublisher = Publisher.newBuilder(serviceTopicName).build()
     }
 
-    fun publish(data: ByteString, onFailureCallback: () -> Unit) {
+    open fun publish(data: ByteString, onFailureCallback: () -> Unit) {
```

Comment:
Is it to make mockito happy? I thought Mockito could mock final methods, but even if it could not, it is better to make our own `PublisherApi` interface anyway, just to make it clear that we do have different implementations (and to let ourselves the liberty to use final methods in implementations)

Reply:
Added PublisherApi interface